CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.4.0
+### Added
+- `void` typehint to `PayseraNormalizationBundle::build` method
```

Comment:
Could we rephrase this to explain why we are making this change? In this case, it should be to fix deprecations.